All GitHub iOS builds failing

The GitLab ones are fine.

```
xcodebuild \
		-workspace ./js/app/ios/Streamplace.xcworkspace \
		-sdk iphoneos \
		-configuration Release \
		-scheme Streamplace \
		-archivePath ./bin/streamplace-v0.7.12-95b01188-ios-release.xcarchive \
		CODE_SIGN_IDENTITY=- \
		AD_HOC_CODE_SIGNING_ALLOWED=YES \
		CODE_SIGN_STYLE=Automatic \
		DEVELOPMENT_TEAM=ZZZZZZZZZZ \
		clean archive | xcpretty \
	&& cd bin \
	&& tar -czvf streamplace-v0.7.12-95b01188-ios-release.xcarchive.tar.gz streamplace-v0.7.12-95b01188-ios-release.xcarchive
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
{ platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:0000FE00-C648B17BC1592925, name:My Mac }
{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
▸ Clean Succeeded
2025-08-15 21:56:01.569 xcodebuild[14775:54490] Writing error result bundle to /var/folders/x7/ch5v91h56_zbvbd1y2f600dm0000gn/T/ResultBundle_2025-15-08_21-56-0001.xcresult
xcodebuild: error: Found no destinations for the scheme 'Streamplace' and action archive.
tar: streamplace-v0.7.12-95b01188-ios-release.xcarchive: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
make: *** [ios] Error 1
```

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -90,11 +90,11 @@ jobs:
           - name: darwin-arm64
             runs-on: macos-15
             cmd: make && make test && make check
-            xcode-version: "16.1.0"
+            xcode-version: "16.4"
           - name: ios
             runs-on: macos-15
             cmd: make ios
-            xcode-version: "16.1.0"
+            xcode-version: "16.4"
           - name: darwin-amd64
             runs-on: macos-13
             cmd: make

